app/views/transactionsAddReqResModal.html

```diff
@@ -95,7 +96,8 @@ <h6 class="subtitle"><em>Received at {{record.response.timestamp | date:'yyyy-MM
           <div ng-if="record.response.body" style="display: inline-block; width: 100%; margin-top: 15px;">
             <h4>Response body</h4>
             <div class="hljsFormat" hljs source="record.response.body"></div>
-            <button style="margin-top: 10px;" class="btn btn-primary pull-right" ng-click="toggleFullView('Response', record.response.body, record.response.headers);">View Full Response Body</button>
+            <transaction-body-downloader transaction-id="transactionId" path="recordPathResponse" style="margin-top: 10px;" class="pull-right"></transaction-body-downloader>
+            <button style="margin-top: 10px;" class="btn btn-primary pull-right" ng-click="toggleFullView('Response', record.response.body, record.response.headers);"><i class="glyphicon glyphicon-resize-full"></i></button>
```

Comment:
Looking awesome Hannes! Do you think it makes sense to include tooltips for the view-fullscreen and download buttons since they are just icons at the moment?